packages/beacon-state-transition/src/phase0/block/processAttesterSlashing.ts

```diff
@@ -1,14 +1,29 @@
 import {allForks, phase0, ValidatorIndex} from "@chainsafe/lodestar-types";
+import {ForkName} from "@chainsafe/lodestar-params";
 
 import {isSlashableValidator, isSlashableAttestationData} from "../../util";
 import {CachedBeaconState} from "../../allForks/util";
 import {isValidIndexedAttestation} from "../../allForks/block";
-import {slashValidator} from "./slashValidator";
+import {slashValidatorAllForks} from "../../allForks/block/slashValidator";
 
 export function processAttesterSlashing(
   state: CachedBeaconState<phase0.BeaconState>,
   attesterSlashing: phase0.AttesterSlashing,
   verifySignatures = true
+): void {
+  processAttesterSlashingAllForks(
+    ForkName.phase0,
+    state as CachedBeaconState<allForks.BeaconState>,
+    attesterSlashing,
+    verifySignatures
+  );
+}
+
+export function processAttesterSlashingAllForks(
```

Comment:
similar to the other place, I think we want to put this in `allForks`, so for common function:
+ `phase0` => `allForks`
+ `altair` => `allForks`

I hope it'd resolve the module loading issue in the spec test